ansible/shell_plugins/docker.py

```diff
@@ -51,13 +62,7 @@ def build_module_command(self, env_string, shebang, cmd, arg_path=None, rm_tmp=N
         pre = ''.join('docker exec {1} mkdir -p {0}; docker cp {0}/. {1}:{0}; '
```

Comment:
>docker cp [](start = 53, length = 9)

If ansible command line has the environment variable `ANSIBLE_KEEP_REMOTE_FILES=0` (this is the default), we still need to remove the temporary file we `docker cp` into the container. I guess you need to override `ShellBase.remove` function.

Reply:
Done